lib/src/signed_video_h26x_auth.c

```diff
@@ -1123,11 +1092,10 @@ validation_is_feasible(const h26x_nalu_list_item_t *item)
   if (item->nalu->is_golden_sei) return false;
   if (!item->nalu->is_hashable) return false;
   if (item->validation_status != 'P') return false;
-
   // Validation may be done upon a SEI.
   if (item->nalu->is_gop_sei) return true;
   // Validation may be done upon the end of a GOP.
-  if (item->nalu->is_first_nalu_in_gop && !item->need_second_verification) return true;
+  if (item->nalu->is_first_nalu_in_gop) return true;
   // Validation may be done upon a hashable NALU right after a SEI. This happens when the SEI was
```

Comment:
The comment states something that is no longer feasible. Remove. Also, remove the while loop below because it was used to find the next NALU.